plugins/module_utils/aci.py

```diff
@@ -1429,10 +1433,25 @@ def get_existing(self):
         that this method can be used to supply the existing configuration when using the get_diff method. The response, status,
         and existing configuration will be added to the self.result dictionary.
         """
-        uri = self.url + self.filter_string
+        if self.suppress_look_back:
```

Comment:
Not sure where to comment because you did not touch the code but is the get_diff function needed when the `suppress_look_back` flag is said?

Reply:
When there is no existing it simply sets config to proposed which is needed even when the flag is set. It won't do any other logic apart from that.